pipelines/DRP.yaml

```diff
@@ -14,3 +14,19 @@ tasks:
       # that's what the (unusual) ci_hsc repo is set up to have.
       connections.photoCalibName: "jointcal"
       useGlobalExternalPhotoCalib: false
+  deblend:
+    class: lsst.pipe.tasks.deblendCoaddSourcesPipeline.DeblendCoaddSourcesMultiTask
+    # Significantly limit the number of sources to deblend, which saves time
+    # in both deblending and measurement
+    config:
+      # Only run on the first 10 parents with between 5-10 children
+      - multibandDeblend.useCiLimits: true
```

Comment:
Are we sure we don't want this to be:

```
- multibandDeblend.ciLimits: 10
```
and have it default to 0 which means no limits?  It all depends on whether we think this will sometimes want to be 100. If the feeling is that CI-or-not-CI is the only use case then I'm fine with leaving it as is.

Reply:
Okay. I've just looked at the other pull request and I see that there are two additional parameters that we are defaulting. I think that's fine and it's the certainty of the comment that is throwing me off.